cpp/include/cugraph/mtmg/resource_manager.hpp

```diff
@@ -153,11 +154,12 @@ class resource_manager_t {
       auto pos = local_rank_map_.find(rank);
       RAFT_CUDA_TRY(cudaSetDevice(pos->second.value()));
 
-      raft::handle_t tmp_handle;
-
+      size_t n_streams{16};
```

Comment:
Why 16?

Reply:
Maybe # of GPUs? (assuming that 1 stream per thread and # threads == # GPUs)

Reply:
Each GPU will have its own pool of streams.  The pool so far is used by different thread ranks copying data to the GPU independently.

Reply:
I've added it as a parameter.